app/src/main/res/values/strings.xml

```diff
@@ -12,4 +12,7 @@
     <string name="error">Error!</string>
     <string name="register">Register</string>
     <string name="fullName">Full Name</string>
+    <string name="boardText">Board</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
These are not needed and are redundant code.

Reply:
Done